ContributedExamples/plot_mht.ipynb

```diff
@@ -4,27 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>As far as I can tell, I don't think any of this code is actually used to produce the figure below (as the figure is a full zonal average)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>